### DISCUSSION TITLE
OpenEphy device couldn't be opened

### DISCUSSION THREAD

**Discussion by LouisonBr** [bonsai-rx/bonsai#1606] [Q&A]
Hi, 

I'm new using Bonsai to synchronize my video and electrophysiology recordings.
I followed the tutorial on OpenEphy website ([https://open-ephys.github.io/acq-board-docs/Tutorials/bonsai-acquisition.html]) but I'm not able to run the workflow. I have this error message "Device couldn't be opened, is one connected?" 

My Open Ephys Acquisition Board and my headstage are well connected, I'm using a new version of the DAQ board with a new FPGA designed by Open Ephys. Can the problem come from here? 

Has someone had this problem and found a solution? 

Thanks a lot, 
Best, 

Louison

**Comment by chiyu1203** [comment]
This sounds like a problem about OpenEphys. Are you sure you are using the right library? There are two OpenEphys library in Bonsai, so I dont know how exactly you set up the workflow. Could you post a screen shot of your workflow?

**Reply by LouisonBr** [reply]
Hi chiyu1203, 

Sorry for this late response, 
I followed the steps described in the Open Ephys website : https://open-ephys.github.io/acq-board-docs/Tutorials/bonsai-acquisition.html
I used the "Bonsai - Ephys Example workflow" and changed the FPGA bitfile as mentioned but it's still not working.
Please find attached a screen shot of the workflow I'm using.

Thank you for your help, 
Best, 

![Picture2](https://github.com/bonsai-rx/bonsai/assets/149614535/b0d6212c-2965-4e0b-85c5-352a0401de5f)
